packages/aquarius/src/bot/commands/roll.js

```diff
@@ -59,7 +59,7 @@ export default async ({ aquarius, analytics }) => {
         const emoji =
           dieType === '100'
             ? `${aquarius.emojiList.get('d10')}${aquarius.emojiList.get('d10')}`
-            : aquarius.emojiList.get(`d${dieType}`);
+            : (aquarius.emojiList.get(`d${dieType}`) || `d${dieType}`);
```

Comment:
Added parentheses not necessary but I can never remember exact precedence rules and they're only one level apart anyways.

Reply:
Looks like the code formatted would have added them too!